src/jit/compiler.h

```diff
@@ -319,6 +402,165 @@ class LclVarDsc
     {
     }
 
+    LiveRangeList* variableLiveRanges;
+
+#if DEBUG
+    LiveRangeBarrier* variableLifeBarrier;
```

Comment:
What is this?

Reply:
It's for printing only the VariableLiveRanges reported/used during a basic block on genCodeForBBList